include/k52/optimization/steepest_descent_method.h

```diff
@@ -3,115 +3,125 @@
 
 #include <boost/shared_ptr.hpp>
 
+#include <k52/common/constants.h>
 #include <k52/optimization/continuous_optimizer.h>
 
 namespace k52
 {
 namespace optimization
 {
 
+//Sources:
+//SteepestDescentMethod, localization method:
+//http://elib.bsu.by/handle/123456789/8648
+//DOC5.NB  ||  MVE - chast'2-2006.ppt
+//Alternative link:
+//https://drive.google.com/folderview?id=0B6NiSXfI55hTMjhTTF85bHpyYzQ&usp=drive_web&tid=0B6NiSXfI55hTSWpYV0g2Z25PejA#list	
+//DOC5.pdf
+//Golden Section Search method:
+//http://vsem-dm.narod.ru/27.html
 
 /// @class SteepestDescentMethod
 /// @brief Realization of Steepest descent optimization method.
 class SteepestDescentMethod : public ContinuousOptimizer
 {
 public:
-	/// @brief Creates instance of HookeJeevesMethod.
-	/// @param alfa - number that defines optimal multiplication step for Golden Section Search method
-	/// @param increment_of_the_argument - number that defines value of gradient step per coordinate
-	/// @param max_iteration_number - maximum number of iterations before computations will be stopped
-	/// @param precision - number that defines accuracy of our computations
-	SteepestDescentMethod(
-		double alpha = 1.6,
-		double increment_of_the_argument = 0.00001,
-		size_t max_iteration_number = 1000,
-		double precision = 0.0001);
-
-	/// @brief  See ICloneable.
-	/// Creates deep clone of an object with resource allocation.
-	/// @return deep clone of an object
-	virtual SteepestDescentMethod* Clone() const;
-
-	virtual std::string get_name() const;
+    /// @brief Creates instance of HookeJeevesMethod.
```

Comment:
HookeJeevesMethod? is it copy-paste error or it means that the SteepestDescentMethod constructor internally creates instance of HookeJeevesMethod?